### DISCUSSION TITLE
JS SDK - Error Handling

### DISCUSSION THREAD

**Discussion by jannismilz** [pocketbase/pocketbase#1162] [Q&A] [answered]
Is there a way to catch errors?
When I'm working with the SDK and even putting it in a try-catch block I can't get the response.. I just get the error itself which isn't helpful at all.

e.g.
```js
try {
   const res = await db.collection('projects').create(formData);
} catch (err) {
   console.log({ err });
}
```
<img width="529" alt="Screenshot 2022-12-02 at 15 22 35" src="https://user-images.githubusercontent.com/67200746/205314121-ff21dce0-d59c-4783-adb2-768c39bb09c8.png">

**Comment by ganigeorgiev** [comment, marked as answer]
You should be able to get error response by accessing `err.data`. All SDK errors are wrapped in `ClientResponseError` object. You could find more info for all available error fields in https://github.com/pocketbase/js-sdk#error-handling.

**Reply by jannismilz** [reply]
Ahh thanks alot. I was searching on the pocketbase docs but ye forgot to check repo 🫢
